debian/postinst

```diff
@@ -78,6 +78,8 @@ case "$1" in
 
       # CACHE_DIR is no longer present or used since 19.1
       rm -rf /var/cache/ubuntu-advantage-tools
+      # machine-access cache files no longer present or used since 20.1
+      rm -rf /var/lib/ubuntu-advantage/private/machine-access*json
```

Comment:
I think we could make this a little tighter (and not recursive):

```suggestion
      rm -f /var/lib/ubuntu-advantage/private/machine-access-*.json
```

Or, alternatively, should we just list out the 5-6 specific files we know will exist, so that this glob can't come back to cause us problems?

Reply:
+1 on dropping the recursive remove. Since this directory is private root read-only, I don't see any exposure we would care to support where other cached json files will exist. If we re-introduce these cache files in the future our packaging will have to take care to drop this cleanup.